src/RelyingParty.js

```diff
@@ -186,6 +191,30 @@ class RelyingParty extends JSONDocument {
     return JSON.stringify(this)
   }
 
+  /**
+   * @description 
+   * Retrieves an existing Relying Party registration for a provider which does 
+   * not support dynamic registration and which requires pre-registration by
+   * some 'out of band' method.
+   *
+   * @param options {Object}
+   * @param idp {string} Key identifying which registration data should be retrieved.
+   * @returns {Promise<Object>} Resolves with the registration response object.
+   */
+  get_registration (options, idp, oob_registration) {
+    return Promise.resolve()
+      .then(() => {
+        return this.registration = oob_registration.getRegistration(idp)
+      })
+      .catch(error => {
+        throw error
+      })
+  }
+
+  serialize () {
```

Comment:
There is already a serialize() function defined in this file. Line 214 adds a duplicate serialize function. (I'm surprised the node vm didn't complain about it! :) )